### PR TITLE
Expose subtitle file size

### PR DESCRIPTION
**Pull request by phyzical**
Also adds an alias for the api under /v1

### CODE REVIEW

File: libs/flask_restx/api.py [hunk 1]
```diff
@@ -873,9 +873,13 @@ def _deferred_blueprint_init(self, setup_state):
...
-        if not setup_state.first_registration:
-            raise ValueError("flask-restx blueprints can only be registered once.")
-        self._init_app(setup_state.app)
+            
+        # https://github.com/flask-restful/flask-restful/issues/509
+        # if not setup_state.first_registration:
+        #     raise ValueError("flask-restx blueprints can only be registered once.")
+        # self._init_app(setup_state.app)
+        if setup_state.first_registration:
+           self._init_app(setup_state.app)
```
Comment (phyzical) [inline libs/flask_restx/api.py:882]: I had to make this change to a lib to allow creating an alias,

@morpheus65535 is this a file that can be modified or is it clobbered? 

If its clobbered any idea how we can make an alias of the api under `/api/v1` ?

File: bazarr/app/server.py [hunk 2]
```diff
@@ -18,6 +18,7 @@
 
 app = create_app()
 ui_bp.register_blueprint(api_bp, url_prefix='/api')
+ui_bp.register_blueprint(api_bp, url_prefix='/api/v1', name="api_alias")
```
Comment (phyzical) [inline bazarr/app/server.py:21]:
```suggestion
```

File: libs/flask_restx/api.py [hunk 3]
```diff
@@ -873,6 +873,7 @@ def _deferred_blueprint_init(self, setup_state):
             setup_state.add_url_rule = MethodType(
                 Api._blueprint_setup_add_url_rule_patch, setup_state
             )
+
```
Comment (phyzical) [inline libs/flask_restx/api.py:876]:
```suggestion
```

**Comment by phyzical** [comment]
@morpheus65535 reverted the api change, i will see if i can make the required around a version lax application side 👍

**Comment by phyzical** [comment]
@morpheus65535 anything still blocking this?

**Comment by morpheus65535** [comment]
> @morpheus65535 anything still blocking this?

Still one comment left

**Review by morpheus65535 — CHANGES_REQUESTED** [review]

File: bazarr/api/swaggerui.py [hunk 4]
```diff
@@ -15,7 +15,8 @@
         "code3": fields.String(),
         "path": fields.String(),
         "forced": fields.Boolean(),
-        "hi": fields.Boolean()
+        "hi": fields.Boolean(),
+        "file_size": fields.Integer(),
```
Comment (phyzical) [inline bazarr/api/swaggerui.py:19]:
```suggestion
        "file_size": fields.Integer()
```

**Review by morpheus65535 — APPROVED** [review]

**Pull request merged**